cordova-plugin-outline/android/java/org/outline/OutlinePlugin.java

```diff
@@ -223,9 +223,12 @@ public void run() {
                   } else if (Action.STOP.is(action)) {
                     stopVpnConnection(connectionId);
                   } else if (Action.IS_REACHABLE.is(action)) {
-                    boolean isReachable =
-                        ShadowsocksConnectivity.isServerReachable(
-                            args.getString(1), args.getInt(2));
+                    boolean isReachable = false;
+                    try {
+                      Shadowsocks.checkServerReachable(args.getString(1), args.getInt(2));
+                      isReachable = true;
+                    } catch (Exception e) {
+                    }
```

Comment:
It looks like you have a logger, so I think you should probably log this exception, if only for debugging.

Reply:
Done.